Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorState.cs

```diff
@@ -242,18 +304,32 @@ private HashSet<int> GetOrCreateExpandedStateInternal()
             return key;
         }
 
+        private static SerializedProperty GetActionMap(SerializedObject serializedObject, int index)
+        {
+            if (serializedObject == null)
+                return null;
+            var property = serializedObject.FindProperty(nameof(InputActionAsset.m_ActionMaps));
+            if (index >= 0 && index < property.arraySize)
+                return property.GetArrayElementAtIndex(index);
+            return null;
+        }
+
         private SerializedProperty GetSelectedActionMap()
         {
-            return serializedObject
-                .FindProperty(nameof(InputActionAsset.m_ActionMaps))
-                .GetArrayElementAtIndex(selectedActionMapIndex);
+            return GetActionMap(serializedObject, selectedActionMapIndex);
         }
 
-        private readonly Dictionary<(string, string), HashSet<int>> m_ExpandedCompositeBindings;
+        private static SerializedProperty GetSelectedAction(SerializedProperty map, int actionIndex)
```

Comment:
Selectors has this method already.

Reply:
Yes but only using a InputActionState and it would be bad practise to pass this before object is fully constructed. I guess we could refactor Selectors to have to GetSelectedAction functions and also refactor the ActionMap ones to accept SerializedObject as an overload...

Reply:
Your suggestion sounds reasonable to me